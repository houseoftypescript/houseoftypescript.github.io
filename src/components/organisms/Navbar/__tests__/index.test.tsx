import { render } from '@testing-library/react';
import { Navbar } from '../index';

describe('Navbar', () => {
  it('renders correctly', () => {
    const button = render(<Navbar title={''} />);
    expect(button.container).toMatchSnapshot();
  });
});
