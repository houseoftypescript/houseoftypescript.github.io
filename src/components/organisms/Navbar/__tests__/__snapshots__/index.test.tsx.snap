// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<div>
  <div
    class="sticky-outer-wrapper"
  >
    <div
      class="sticky-inner-wrapper"
      style="position: relative; z-index: 100; transform: translate3d(0,0px,0);"
    >
      <nav
        class="bg-white"
      >
        <div
          class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
        >
          <div
            class="py-8"
          >
            <div
              class="flex justify-between items-center"
            >
              <div
                class="flex items-center gap-8"
              >
                <h1
                  class="text-2xl"
                >
                  <a
                    class="cursor-pointer"
                  />
                </h1>
                <div
                  class="hidden md:flex items-center gap-8"
                />
              </div>
              <div
                class="flex items-center gap-8"
              >
                <a
                  class="cursor-pointer text-emerald-500 font-bold"
                >
                  Get Started
                </a>
                <div
                  class="block md:hidden"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="MenuIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
      </nav>
    </div>
  </div>
</div>
`;
